### ISSUE TITLE
Adding timestamps error (key too long)

### ISSUE THREAD

**Issue by MikeDabrowski** [sequelize/cli#627]
## What were you doing?

```js
node_modules/.bin/sequelize db:migrate:schema:timestamps:add
```

## What do you expect to happen?
I expected it to add timestamps to Meta table as instructions says

## What is actually happening?
But the output was `ERROR: Specified key was too long; max key length is 767 bytes
`

__Dialect:__ mysql
__Database version:__ 10.1.30-MariaDB-0ubuntu0.17.10.1
__Sequelize CLI version:__ 4.0.0
__Sequelize version:__ 4.35.2

Sequelize CLI [Node: 9.6.1, CLI: 4.0.0, ORM: 4.35.2]

**Comment by kyleguate** [comment]
This is happening for me too:

**Dialect**: mysql
**Database version**: MySQL 5.6.34
**Sequelize CLI version**: 4.0.0
**Sequelize version**: 4.36.1

```
ERROR: Specified key was too long; max key length is 767 bytes
```

**Comment by MikeDabrowski** [comment]
I finally fixed this by updating to latest mariadb. Unique on create
varchar(255) is making it over 767.

**Comment by kyleguate** [comment]
From what I've read the default value is changed in 5.7 to enable `innodb_large_prefix`. I'll have to see if we can get enable that value on our db.

https://github.com/YOURLS/YOURLS/issues/2115
https://dev.mysql.com/doc/refman/5.7/en/innodb-parameters.html#sysvar_innodb_large_prefix

This would probably be worth noting in the docs somewhere.

**Comment by kyleguate** [comment]
Not sure if this would cause problems, but it looks like on the `mysql` side that the default changes introduced in 5.7.7 may not behave as expected (it depends on the table row format)

From [here](https://dev.mysql.com/doc/refman/5.7/en/innodb-parameters.html#sysvar_innodb_large_prefix)

> For tables that use REDUNDANT or COMPACT row format, this option does not affect the permitted index key prefix length.

> innodb_large_prefix is enabled by default in MySQL 5.7. This change coincides with the default value change for innodb_file_format, which is set to Barracuda by default in MySQL 5.7. Together, these default value changes allow larger index key prefixes to be created when using DYNAMIC or COMPRESSED row format. If either option is set to a non-default value, index key prefixes larger than 767 bytes are silently truncated.

I wonder if the value being truncated affects how sequelize manages the table

**Comment by MikeDabrowski** [comment]
I solved this by upgrading mariadb to 10.3
